src/modules/auth/index.js

```diff
@@ -122,4 +137,28 @@ export default class AuthModule extends Module {
 		return true;
 	}
 
+
+	async _isUserAuthorized() {
+		let resp = false;
+		// TODO: Solve race condition when obtaining tenant from redux store
+		const state = this.Store.getState();
```

Comment:
@ateska Here I have a problem to get the current tenant from the store because of race condition. I cannot await it here, so I tried to dispatch it with promise in tenant reducer, but I have not been successful, so temporarily I solved it with the solution that when `activeTenant` (current) is null, then it will take the current tenant from the parameters of the browser. Do you have a hint, how to solve this issue?

Reply:
The promise seems to be a good idea. Let's address this in a new MR.